onnxruntime/core/providers/cuda/cuda_execution_provider_info.h

```diff
@@ -13,15 +13,20 @@
 namespace onnxruntime {
 // Information needed to construct CUDA execution providers.
 struct CUDAExecutionProviderExternalAllocatorInfo {
-  void* alloc{nullptr};
-  void* free{nullptr};
+  const void* alloc{nullptr};
+  const void* free{nullptr};
 
   CUDAExecutionProviderExternalAllocatorInfo() {
```

Comment:
this constructor should be removed, since we always expect a value of alloc and free pointer

Reply:
I think we need that default constructor because of this line CUDAExecutionProviderExternalAllocatorInfo external_allocator_info{} and this line is needed because of CUDAExecutionProviderInfo{}